src/main/java/frc/robot/Subsystems/WincherClimber.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.Subsystems;
+
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+
+public class WincherClimber extends MotoredGenericSubsystem {
+    private static final double MIN_SPEED = -0.6;
+    private static final double MAX_SPEED = 0.6;
+    private Level magnetLevel = Level.LOWER;
+    private static WincherClimber instance;
+    private final DigitalInput HallEffect;
+
+    enum Level {UPPER, LOWER, MIDDLE}
+
+    public WincherClimber(WPI_VictorSPX rightWinch, WPI_VictorSPX leftWinch, DigitalInput HallEffect) {
+        super(MIN_SPEED, MAX_SPEED, "wincherClimber", rightWinch, leftWinch);
+        this.HallEffect = HallEffect;
+    }
+
+    public static WincherClimber getInstance() {
+        if (instance == null) {
+            instance = new WincherClimber(new WPI_VictorSPX(-1), new WPI_VictorSPX(-1),
+                    new DigitalInput(-1));
+        }
+        return instance;
+    }
+
+
+    @Override
+    public boolean canMove(double speed) {
```

Comment:
This method should only check if this subsystem can move. It shouldn't do anything else (such as changing the `magnetLevel`). Do this in another method.